src/js/components/RadioButton/doc.js

```diff
@@ -116,6 +116,12 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '24px',
   },
+  'radioButton.hover.background.color': {
+    description: `The background color of the Box surrounding the RadioButton
+    when hovered over.`,
+    type: 'string | { dark: string, light: string }',
+    defaultValue: '{dark: undefined, light: undefined}',
```

Comment:
This actually defaults to an object (with undefined values for its keys), we should default to undefined.
```suggestion
    defaultValue: 'undefined',
```